cachito/workers/tasks.py

```diff
@@ -47,15 +47,16 @@ def fetch_app_source(url, ref):
 
 
 @app.task
-def fetch_gomod_source(archive_path):
+def fetch_gomod_source(archive_path, copy_cache=None):
     """
     Resolve and fetch gomod dependencies for given app source archive.
 
     :param str archive_path: the full path to the application source code
+    :param str copy_cache: path to copy artifacts from gomod cache
```

Comment:
`copy_cache` and `archive_path` should ideally be relative paths instead of absolute paths. With this current implementation, the API has to know where the storage mount is in all the workers. Where the storage mount should be an implementation detail, and the API doesn't actually need to know about `archive_path`, it just needs to know the worker has it stored somewhere.

Reply:
Yup. I'll try to make that work.

Reply:
Thanks for the suggestion. Made the change to use relative paths, much better.